config/settings.py

```diff
@@ -0,0 +1,196 @@
+"""
+Django settings for kitab bazar project.
+
+Generated by 'django-admin startproject' using Django 4.0.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get(
+    'SECRET_KEY',
+    'django-insecure-2s$6zbn2r_0=pao&lf3wz8q%9s*hsi=*)ng=llpkr&3h=#2#85'
+)
```

Comment:
Let's use django-environ.

Example:
  Usages: https://github.com/the-deep/server/blob/feature/new-deployment-changes/deep/settings.py#L22-L79
  pyproject: https://github.com/the-deep/server/blob/feature/new-deployment-changes/pyproject.toml#L69